project_3.html

```diff
@@ -0,0 +1,470 @@
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Implement the <code class="pj-code language-arduino">isEven()</code> function as described by the project requirements. Verify your code by writing tests in <code class="pj-code language-arduino">main()</code>.
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 1" content="When dividing an even number by 2, there is no remainder. However, dividing an odd number by 2 results in a remainder of 1."></script>
+					</div>
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 2" content="The modulus % operator returns the remainder of a division operation."></script>
+					</div>
+				</li>
+				<li class="list-group-item">
+					Verify your code by writing tests in <code class="pj-code language-arduino">main()</code>.
+				</li>
+			</ol>
+
+			<h3 class="mb-3">Checkpoint 4: Number Power</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Implement the <code class="pj-code language-arduino">pow2()</code> function as described by the project requirements.
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 1" content="Remember that raising the base to the power of an exponent is the same as multiplying the base by itself the number of times equal to the exponent."></script>
+					</div>
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 2" content="Consider using a for loop."></script>
+					</div>
+				</li>
+				<li class="list-group-item">
+					Verify your code by writing tests in <code class="pj-code language-arduino">main()</code>.
+				</li>
+			</ol>
+
+			<h3 class="mb-3">Checkpoint 5: Take the Average</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Implement the <code class="pj-code language-arduino">avg()</code> function as described by the project requirements.
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 1" content='Recall the <a href="https://en.wikipedia.org/wiki/Mean">formula</a> for calculating the mean of a number set.''></script>
+					</div>
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 2" content="Consider using a for loop."></script>
+					</div>
+				</li>
+				<li class="list-group-item">
+					Verify your code by writing tests in <code class="pj-code language-arduino">main()</code>.
+				</li>
+			</ol>
+
+
+			<h3 class="mb-3">Checkpoint 6: Fibonacci Sequence</h3>
+			<ol class="mb-5 list-group list-group-numbered">
+				<li class="list-group-item">
+					Implement the <code class="pj-code language-arduino">fib()</code> function as described by the project requirements.
+					<div>
+						<script src="./js/collapsible.js" title="Click to Reveal Hint 1" content="Recall the pattern of the Fibonacci Sequence."></script>
```

Comment:
Please include a link to the wikipedia page included in the original project doc